beacon-chain/db/filesystem/blob.go

```diff
@@ -242,6 +242,21 @@ func (bs *BlobStorage) Get(root [32]byte, idx uint64) (blocks.VerifiedROBlob, er
 	return verification.BlobSidecarNoop(ro)
 }
 
+// GetColumn retrieves a single BlobSidecar by its root and index.
+func (bs *BlobStorage) GetColumn(root [32]byte, idx uint64) (*ethpb.DataColumnSidecar, error) {
+	expected := blobNamer{root: root, index: idx}
```

Comment:
Should we rename `blobNamer` into something more generic, which could fit both blobs and columns?

Reply:
Or maybe not. If we eventually end up to totally remove blob and keep columns, then we can rename types only at this time.

Reply:
I think its something we can do in the future, but for the P.O.C , it is fine to reuse all the blob storage methods without changing the name to be more generic.